auth/email_action_links_test.go

```diff
@@ -40,6 +40,7 @@ var testActionCodeSettings = &ActionCodeSettings{
 	AndroidPackageName:    "com.example.android",
 	AndroidInstallApp:     true,
 	AndroidMinimumVersion: "6",
+	ReturnOobLink:         true,
```

Comment:
Can we remove this in the base testActionCodeSettings struct? We need a test case to verify that if this field is not specified, returnOobLink is still set to true, due to sdk implementation.

I suggest adding a new test case in  the newly added test, see my other comment. Thanks!